Classes/Private/Wrapper/OCTToxAV.m

```diff
@@ -258,7 +388,16 @@ void callstateCallback(ToxAV *cToxAV,
                        uint32_t state,
                        void *userData)
 {
-    // To Do..
+    OCTToxAV *toxAV = (__bridge OCTToxAV *)userData;
+
+    dispatch_async(dispatch_get_main_queue(), ^{
+        DDLogCInfo(@"%@: callstateCallback from friend %d with state: %d", cToxAV, friendNumber, state);
+
+
+        if ([toxAV.delegate respondsToSelector:@selector(toxAV:callStateChanged:friendNumber:)]) {
+            [toxAV.delegate toxAV:toxAV callStateChanged:(OCTToxAVCallState)state friendNumber:friendNumber];
```

Comment:
But if it's a bitmask how would you do that do that in a switch? I apologize if it's not too obvious for me.